README.md

```diff
@@ -1,2 +1,26 @@
 # LinQuadOptInterface.jl
-Intermediate wrapper for MOI of CPLEX, Gurobi and Xpress
+
+LinQuadOptInterface (LQOI) is a intermediate wrapper designed to bridge low-level Integer Linear and Quadratic solvers. It provides access to many MathOptInterface functionalities mainly related to problem modifications: 
```

Comment:
Designed to make it easier for low-level [...] solvers to implement the MOI interface.